quota/v1/types.go

```diff
@@ -40,9 +40,13 @@ type ClusterResourceQuotaSpec struct {
 // the project must match both restrictions.
 type ClusterResourceQuotaSelector struct {
 	// LabelSelector is used to select projects by label.
+	// +optional
+	// +nullable
```

Comment:
Drop the `nullable` optional should be sufficient from what I've looked.

Reply:
Based on what @damemi told me `optional` is only informational and `nullable` is the one we want.